doc/getting_started/index.rst

```diff
@@ -330,38 +330,26 @@ Set up Reference UOS
 Device Manager memory allocation mechanism
 ==========================================
 
-There are two Device Manager memory allocation mechanisms available:
+The ACRN Device Manager (DM) virtual memory allocation uses the HugeTLB mechanism.
+(You can read more about `HugeTLB in the linux kernel <https://linuxgazette.net/155/krishnakumar.html>`
```

Comment:
need to have an underscore at the end of the link:

    `HugeTLB in the linux kernel <https://linuxgazette.net/155/krishnakumar.html>`_

Reply:
done.
Thx